albumentations/core/transforms_interface.py

```diff
@@ -163,16 +172,16 @@ def add_targets(self, additional_targets):
         self._additional_targets = additional_targets
 
     @property
-    def targets_as_params(self):
+    def targets_as_params(self) -> List:
         return []
 
-    def get_params_dependent_on_targets(self, params):
+    def get_params_dependent_on_targets(self, params: Dict[str, Any]):
         raise NotImplementedError(
             "Method get_params_dependent_on_targets is not implemented in class " + self.__class__.__name__
         )
 
     @classmethod
-    def get_class_fullname(cls):
+    def get_class_fullname(cls) -> str:
         return get_shortest_class_fullname(cls)
 
     def get_transform_init_args_names(self):
```

Comment:
I am getting the following mypy error:

```
albumentations\augmentations\crops\transforms.py:812: error: Return type "Sequence[str]" of "get_transform_init_args_names" 
incompatible with return type "Tuple[str, ...]" in supertype "BasicTransform"
```

Seems to be because the code below has the return type-hint ```-> Sequence[str]```. Shall I change the type hint here to ```-> Tuple[str, ...]``` as well?

https://github.com/albumentations-team/albumentations/blob/2cefc162d7c605f76462e3cd5ce8c3c5117a44af/albumentations/augmentations/crops/transforms.py#L812-L822

Reply:
Yes, just change it to `Tuple`